src/core/dbc-gateway/graphql-fetcher.ts

```diff
@@ -9,9 +9,12 @@ export const fetcher = <TData, TVariables>(
     // First version is with a library token.
     const libraryToken = getToken(TOKEN_LIBRARY_KEY);
 
-    if (!libraryToken) {
-      throw new Error("Library token is missing!");
-    }
```

Comment:
I think we should not remove the missing token error handling.
You could argue if an error should be thrown or logged or whatever. But I think we need error handling because the FBI services will not work without a token.
I have solved the same problem (about Cypress failing because the token is not present) by setting the token in sessionStorage which we use in storybook to set tokens that need to be accessable:
https://github.com/danskernesdigitalebibliotek/dpl-react/pull/83/files#diff-2681eddd9ccddb0757dced492f1e67c5065af0d811d13c6d071a7b9d09124753R82-R84

Reply:
Disrtegard this. We keep it like this ;)

Reply:
talked about this live :)